Getting-Started.md

```diff
@@ -709,4 +853,58 @@ Kibiter then automatically generates a query as shown below.
 
 > :warning: **If you change the structure or delete it** you won't see the menu.
 
-> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
\ No newline at end of file
+> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
+
+#### Dump a mapping/data from an index [&uarr;](#dump-data/mapping-)
+
+**elasticdump installation**
+
+```
+sudo apt install npm
+sudo -i
+npm install elasticdump -g
+
+```
+
+![elasticdump_installation](https://user-images.githubusercontent.com/53489999/77806917-23786500-70ac-11ea-9d09-dbdb38caf6aa.png)
+
+CTRL+D to logout from root user
+
+***
+
+> Use the 'type' argument to either fetch data or mapping.
+
+
+**To get data from the index**
+
+```
+elasticdump --input=http://localhost:9200/git_chaoss/ --output=git_data.json --type=data
+```
+
+![elasticdump_data](https://user-images.githubusercontent.com/53489999/77806813-d4cacb00-70ab-11ea-9a58-dcb3c7715e9d.png)
+
+**To get mapping from the index**
+
+```
+elasticdump --input=http://localhost:9200/git_chaoss/ --output=git_mapping.json --type=mapping
+```
+
+![elasticdump_mapping](https://user-images.githubusercontent.com/53489999/77806637-40f8ff00-70ab-11ea-8fe5-766fbde93fcf.png)
+
+This will generate output files 
```

Comment:
I would remove the content after this line (included) since it doesn't give additional information to the user that wants to dump an index/mapping. WDYT?

Reply:
Yes..I agree. Will remove it. Thanks!